wave_matching/config/groundSegmentation.yaml

```diff
@@ -0,0 +1,17 @@
+rmax:                     100 #maximum radius of points to consider
+num_maxbinpoints:         200 #max Number of Points in each Bin
+num_seedpoints:           10  #Number of seed points to use in GP
+num_ang_bins:             72  #Number of angular bins to use
+num_lin_bins:             200 #Number of linear bins to use
+
+# GP Kernel hyperparameters
+gp_lengthparameter:       10
+gp_covariancescale:       1
+gp_modelnoise:            0.3
+gp_groundmodelconfidence: 5
+gp_grounddataconfidence:  5
+
+gp_groundthreshold:       0.3 #Ground Height Threshold
+robotheight:              1.2 #Used for drivable surface detections I think
+seeding_maxrange:         50  #Maximum range for seed points
```

Comment:
What is a "seed point"? Is there additional documentation? At least a reference to a paper that defines these terms?

Reply:
I think it is following the approach in https://link.springer.com/article/10.1007/s10846-013-9889-4. I think @arunxdas 's thesis also mentions it.

Reply:
Thanks, add to code somewhere

Reply:
Let's wait until Arun confirms that it's the right one.